portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,273 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+.background-block {
+    background-color: #e897c3 ;
+    margin-top: -34px;
+    padding-bottom: 400px;
+    width: 100%;
 }
 
-#greeting-container {
-  margin-top: 20px;
+.logo {
+    left: 15px;
+    position: absolute;
+    top: 105px;
 }
+
+.icon {
+    margin-top: 15%;
+}
+
+.bitmoji-headshot {
+    display: block;
+    margin: 30px auto;
+}
+
+.real-headshot {
+    margin-left: 200px;
+    margin-top: 100px;
+}
+
+h1 { 
+    font: normal normal 700 40px/42.4px 'Rockwell Extra Bold', 'Rockwell Bold', monospace; 
+    margin-top: 115px;
+    white-space: nowrap;
+}
+
+h2 { 
+    color: white;
+    font: normal normal 700 30px/32.4px 'Rockwell Extra Bold', 'Rockwell Bold';
+    padding-top: 115px;
+    white-space: nowrap;
+}
+
+h3 { 
+    font: normal normal 100 20px/21.4px Rockwell, 'Courier Bold', Courier, Georgia, Times, 'Times New Roman', serif;
+    margin-top: 30px;
+    white-space: nowrap;
+}
+
+h4 { 
+    color: black;
+    font: normal normal 100 25px/21.4px Rockwell, 'Courier Bold', Courier, Georgia, Times, 'Times New Roman', serif;
+    margin-top: 30px;
+    white-space: nowrap;
+}
+
+p.white {   
```

Comment:
Avoid qualifying ID and class names with type selectors.

https://google.github.io/styleguide/htmlcssguide.html#Type_Selectors